pkg/controller/volume/attachdetach/reconciler/reconciler.go

```diff
@@ -154,6 +155,15 @@ func (rc *reconciler) hasOutOfServiceTaint(nodeName types.NodeName) (bool, error
 	return false, nil
 }
 
+// isHealthy returns true if the node looks healthy.
+func (rc *reconciler) isHealthy(nodeName types.NodeName) (bool, error) {
+	node, err := rc.nodeLister.Get(string(nodeName))
+	if err != nil {
```

Comment:
do we want to check whether node object still exist or not?

Reply:
It is an informer, so it won't error on network hiccups. I've never seen a temporal errors returned.